### PR TITLE
Automatically load card filters in the CardFilter directory

### PR DESCRIPTION
**Pull request by medlion**
This does use functionality not available in Symfony >4.0

### CODE REVIEW

**Comment by medlion** [comment]
```
Loading cards from API...
Loading lists from API...
Total Cards: 3 Filtered Cards: 3
Options:
  [0] Add Filter
  [1] Remove Filter
  [2] Print Cards
  [3] Refresh Cards
  [4] Exit
 > 0
Select Filter:
  [0] Activity Since
  [1] Has Member
  [2] Has No Labels
  [3] Has No Members
  [4] In List
  [5] Label Colour
  [6] NOT
  [7] OR
  [8] <- Back
```

File: src/AppBundle/Helper/CardFilter/CardFilterFactory.php [hunk 1]
```diff
@@ -103,4 +95,17 @@ public function getFilterClasses()
...
         return $this->filterClasses;
     }
-}
\ No newline at end of file
+
+    private function getFilesInDirectoryAndSubdirectories (string $path = __DIR__)
+    {
+        $arrayOfClasses = [];
+        foreach (array_keys(\Composer\Autoload\ClassMapGenerator::createMap($path)) as $class) {
+            if ($class === AbstractCardFilter::class || $class === CardFilterInterface::class || $class === __CLASS__) {
```
Comment (mostertb) [inline src/AppBundle/Helper/CardFilter/CardFilterFactory.php]: Is it possible to simplify this into using `class_implements()`  like you've done in the `validateFilterClasses()` function and then use the `isAbstract()` function from ReflectionClass (https://www.php.net/manual/en/reflectionclass.isabstract.php) to help futureproof us for if additional abstract classes are added?

**Comment by mostertb** [comment]
This is looking good 👍 

I've added one comment. 

Could you also please update the `CONTRIBUTING.md` as per #3 ?

**Comment by mostertb** [comment]
Thanks for the PR!

**Pull request merged**